src/components/SummaryBox.jsx

```diff
@@ -0,0 +1,106 @@
+            <h1 className="sectionTitle"> {sectionTitle} </h1>
+          </div>
+          <div id="bar" />
+          <div className="outer-circle red-b" id="left-circle">
+            <div className="inner-circle">
+              <p className="circle-text red-c" id="circle-left-text">
+                Biased
+              </p>
+            </div>
+          </div>
+          <div className="outer-circle">
+            <div className="inner-circle">
+              <p className="circle-text" id="circle-center-text">
+                Somewhat Biased
+              </p>
+            </div>
+          </div>
+          <div className="outer-circle">
+            <div className="inner-circle">
+              <p className="circle-text" id="circle-right-text">
+                Not Biased
+              </p>
+            </div>
+          </div>
+          <div id="rectangle" className="red-b" />
+        </div>
+      );
+    } else if (val == 1) {
+      // MAYBE
+      return (
+      <div className="figmaCard">
+        <div className="boxboxbox">
+          <h1 className="sectionNum"> Section {sectionNumber} </h1>
+          <h1 className="sectionTitle"> {sectionTitle} </h1>
+        </div>
+        <div id="bar"></div>
+        <div className="outer-circle" id="left-circle">
+          <div className="inner-circle">
+            <p className="circle-text" id="circle-left-text">Biased</p>
+          </div>
+        </div>
+        <div className="outer-circle yellow-b">
+          <div className="inner-circle">
+            <p className="circle-text yellow-c" id="circle-center-text">Somewhat Biased</p>
+          </div>
+        </div>
+        <div className="outer-circle">
+          <div className="inner-circle">
+            <p className="circle-text" id="circle-right-text">Not Biased</p>
+          </div>
+        </div>
+        <div id="rectangle" className="yellow-b"></div>
+      </div>
+    );
+    } else {
+      // NO
+      return (
+      <div className="figmaCard">
+        <div className="boxboxbox">
+          <h1 className="sectionNum"> Section {sectionNumber} </h1>
+          <h1 className="sectionTitle"> {sectionTitle} </h1>
+        </div>
+        <div id="bar"></div>
+        <div className="outer-circle" id="left-circle">
+          <div className="inner-circle">
+            <p className="circle-text" id="circle-left-text">Biased</p>
+          </div>
+        </div>
+        <div className="outer-circle">
+          <div className="inner-circle">
+            <p className="circle-text" id="circle-center-text">Somewhat Biased</p>
+          </div>
+        </div>
+        <div className="outer-circle green-b">
+          <div className="inner-circle">
+            <p className="circle-text green-c" id="circle-right-text">Not Biased</p>
+          </div>
+        </div>
+        <div id="rectangle" className="green-b"></div>
+      </div>
+    );
+    }
+  }
+}
+
+export default SummaryBox;
```

Comment:
It seems like there's some duplicated html elements in here.
Can you nest the selected child like:
```
<div className="figmaCard">
  <div className="boxboxbox">
    <h1 className="sectionNum"> Section {sectionNumber} </h1>
    <h1 className="sectionTitle"> {sectionTitle} </h1>
  </div>
  <div id="bar" />
  <div className={`outer-circle ${val == 0? 'red-b' : ''}`}>
    <div className="inner-circle">
      <p className={`circle-text ${val == 0? 'red-c' : ''}`}>
    ...
```

Reply:
This is a way of passing in css classNames so your if statements generate the same html elements but with different styles.